packages/kcms/src/match/controller.ts

```diff
@@ -0,0 +1,48 @@
+import { z } from '@hono/zod-openapi';
+import { Result } from '@mikuroxina/mini-fn';
+import { SnowflakeIDGenerator } from '../id/main';
+import { TeamID } from '../team/models/team';
+import { PostMatchRunResultRequestSchema } from './adaptor/validator/match';
+import { FinishState } from './model/runResult';
+import { CreateRunResultService } from './service/createRunResult';
+type MatchResults = {
+  teamID: TeamID;
+  points: number;
+  goalTimeSeconds: number;
+  finishState: FinishState;
+}[];
+export class Controller {
+  private readonly createRunResult: CreateRunResultService;
+  constructor() {
+    this.createRunResult = new CreateRunResultService(
+      new SnowflakeIDGenerator(1, () => BigInt(new Date().getTime()))
+    );
+  }
+  async create(args: z.infer<typeof PostMatchRunResultRequestSchema>) {
```

Comment:
戻り値の型付けがされていません。APIから考えると
```suggestion
  async create(
    args: z.infer<typeof PostMatchRunResultRequestSchema>
  ): Promise<Result.Result<Error, void>> {
```
でしょうか？(レスポンスボディが存在しないため)